orcestra_book/plans/HALO-20240903a.md

```diff
@@ -71,55 +71,59 @@ atr_radius = 38e3*1.852
 c_atr_nw = LatLon(17.433,-23.500, label = "c_atr")
 c_atr_se = LatLon(16.080,-21.715, label = "c_atr")
 
+c_new = LatLon(15.580,-22.1, label = "c_atr")
 # Define dates for flight
 
 flight_time   = datetime(2024, 9, 3, 12, 0, 0)
 flight_index = f"HALO-{flight_time.strftime('%Y%m%d')}a"
 
 # Load satellite tracks 
 
-ec_fcst_time  = "2024-09-01"
-ec_track = orcestra.sat.SattrackLoader("EARTHCARE", ec_fcst_time, kind="PRE") \
-    .get_track_for_day(f"{flight_time:%Y-%m-%d}")\
-    .sel(time=slice(f"{flight_time:%Y-%m-%d} 14:00", None))
+if (True):
+    ec_fcst_time  = "2024-09-01"
+    ec_track = orcestra.sat.SattrackLoader("EARTHCARE", ec_fcst_time, kind="PRE") \
+        .get_track_for_day(f"{flight_time:%Y-%m-%d}")\
+        .sel(time=slice(f"{flight_time:%Y-%m-%d} 14:00", None))
```

Comment:
I strongly recommend to remove this change. It doesn't do anything to the flight plan, apart from confusing readers.

```suggestion
ec_fcst_time  = "2024-09-01"
ec_track = orcestra.sat.SattrackLoader("EARTHCARE", ec_fcst_time, kind="PRE") \
    .get_track_for_day(f"{flight_time:%Y-%m-%d}")\
    .sel(time=slice(f"{flight_time:%Y-%m-%d} 14:00", None))
```